crates/astria-core/src/execution/v1alpha2/mod.rs

```diff
@@ -2,9 +2,107 @@ use prost_types::Timestamp;
 
 use crate::{
     generated::execution::v1alpha2 as raw,
+    sequencer::v1alpha1::{
+        IncorrectRollupIdLength,
+        RollupId,
+    },
     Protobuf,
 };
 
+// An error when transforming a [`raw::Block`] into a [`Block`].
```

Comment:
```suggestion
/// An error when transforming a [`raw::Block`] into a [`Block`].
```
to make this into a doccomment